src/components/shared/transactionsTable/index.js

```diff
@@ -73,7 +73,7 @@ const TransactionsTable = ({
         clearFilter, clearAllFilters, filters, formatters, t,
       }}
       />
-      <BoxContent className={styles.content}>
+      <BoxContent className={`${styles.content} transaction-results`}>
```

Comment:
`transaction-results` was added as a selector class for cypress though it wasn't used any were in the PR

Reply:
@eniolam1000752 This was done for uniformity to ensure that such selector can be used in any future test where a specific value is to be checked for in the table. For example, `cy.get(ss.transactionsTable).contains('lskgpfj26e3uogvr5wbono75hzn5myds4pncv3zm8').should('exist')`. However I can remove it for the time being